### ISSUE TITLE
Add Swift Package Manager support

### ISSUE THREAD

**Issue by fmoraes74** [bmichotte/RegexUtil#1]
Can you add `Package.swift` to enable SPM support in Xcode?

**Comment by bmichotte** [comment]
Yes, sure. I’ll try to do it but in the meantime, you can PR it

**Comment by fmoraes74** [comment]
I think all you need to do is add a Package.swift such as:

```
// swift-tools-version:5.0
// The swift-tools-version declares the minimum version of Swift required to build this package.
import PackageDescription

let package = Package(
    name: "FMDB",
    products: [
        // Products define the executables and libraries produced by a package, and make them visible to other packages.
        .library(name: "FMDB", targets: ["FMDB"]),
    ],
    dependencies: [
        // Dependencies declare other packages that this package depends on.
        // .package(url: /* package url */, from: "1.0.0"),
    ],
    targets: [
        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
        .target(
            name: "FMDB",
            dependencies: [],
            path: "src/fmdb",
            publicHeadersPath: "."),
    ]
)
```

**Comment by bmichotte** [comment]
Here you go, you should now be able to add the lib with SwiftPM 

Closed by bcea6b4779e505ef54cbdc09096f5dd7e0d1be86

**Comment by fmoraes74** [comment]
Thanks!
